.github/workflows/build.yml

```diff
@@ -461,44 +461,77 @@ jobs:
           git config --global user.name "github-actions[bot]"
           git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
 
-      - name: Release Charts
+      - name: Prepare Chart Release and index entry
         if: ${{ needs.setup.outputs.run_build == 'true' }}
         env:
           BOT_TOKEN: ${{ secrets.BOT_TOKEN }}
-          GITHUB_REF: ${{ github.ref }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           REPORT_CONTENT: ${{ needs.chart-verifier.outputs.report_content }}
-          CHART_ENTRY_NAME: ${{ needs.chart-verifier.outputs.chart_entry_name }}
           REDHAT_TO_COMMUNITY: ${{ needs.chart-verifier.outputs.redhat_to_community }}
           WEB_CATALOG_ONLY: ${{ needs.chart-verifier.outputs.web_catalog_only }}
           OCP_VERSION_RANGE: ${{ steps.get-ocp-range.outputs.ocp-version-range }}
-        id: release-charts
+        id: prepare-chart-release
         run: |
-          tar zxvf ./scripts/dependencies/helm-chart-releaser/chart-releaser_1.2.0_linux_amd64.tar.gz
-          sudo cp -f cr /usr/local/bin/cr
-          INDEX_BRANCH=$(if [ "${GITHUB_REF}" = "refs/heads/main" ]; then echo "refs/heads/gh-pages"; else echo "${GITHUB_REF}-gh-pages"; fi)
-          CWD=`pwd`
           cd pr-branch
           ../ve1/bin/chart-repo-manager \
             --repository=${{ github.repository }} \
-            --index-branch=${INDEX_BRANCH} \
             --api-url=${{ github.event.pull_request._links.self.href }} \
-            --pr-number=${{ github.event.number }}
-          cd ${CWD}
 
-      - name: Release
+      # Upload the report file, potentially paired with a public key and, if provided, the chart's tarball and its prov file.
+      # Only the report file is always included.
+      # The release tag format is <organization_name>-<chart_name>-<chart_version>
+      - name: Create GitHub release
         if: ${{ needs.chart-verifier.outputs.web_catalog_only == 'False' }}
-        uses: softprops/action-gh-release@v0.1.12
-        continue-on-error: true
```

Comment:
We no longer want to continue on error?

Reply:
Good point (BTW I'm not always happy with the way we extensively use `continue-on-error` and fail later. I think it makes debugging more complicated and is not always necessary. Maybe a design discussion we need to have, I'll try to document my annoyance in a separate issue one day :) ).

In this particular case I don't see the benefit of `continue-on-error`. Let's take a look at the following steps:
- `Update Helm repository index` will fail anyway if the release doesn't exist. There is a check within the `update-index` script that the release exists for this chart.
- `Retrieve PR comment for metrics` and `Add metrics` have the `always()` condition, so they will run no matter what.

Reply:
CI didn't fail, so we can continue without it for now. Typically, the reason I see it is so that we can comment on the PR, but at this point in the workflow, I don 't think there's any commenting that comes later so we should be in the  clear.